kyaml/fieldmeta/fieldmeta.go

```diff
@@ -174,6 +172,22 @@ func (fm *FieldMeta) Write(n *yaml.RNode) error {
 	return nil
 }
 
+// WriteV1Setters is the v1 setters way of writing setter definitions
+// TODO: pmarupaka - remove this method after migration
+func (fm *FieldMeta) WriteV1Setters(n *yaml.RNode) error {
+	if !isExtensionEmpty(fm.Extensions) {
```

Comment:
nit: true clause first - easier to read if not notted :-)

Reply:
I missed that. Thanks for pointing. Addressed.